tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/impl/descartes_motion_planner.hpp

```diff
@@ -170,23 +158,15 @@ tesseract_common::StatusCode DescartesMotionPlanner<FloatType>::solve(PlannerRes
 
   auto tStart = boost::posix_time::second_clock::local_time();
 
-  const auto dof =
-      config_->tesseract->getFwdKinematicsManagerConst()->getFwdKinematicSolver(config_->manipulator)->numJoints();
+  const auto dof = config_->joint_names.size();
   descartes_light::Solver<FloatType> graph_builder(dof);
   if (!graph_builder.build(
           config_->samplers, config_->timing_constraint, config_->edge_evaluator, config_->num_threads))
   {
     CONSOLE_BRIDGE_logError("Failed to build vertices");
     for (const auto& i : graph_builder.getFailedVertices())
-    {
-      const Waypoint::Ptr& wp = config_->waypoints[i];
-      if (wp->getType() == WaypointType::CARTESIAN_WAYPOINT)
-      {
-        CartesianWaypoint::ConstPtr cwp = std::static_pointer_cast<const CartesianWaypoint>(wp);
-        config_->kinematics->analyzeIK(cwp->getTransform().cast<FloatType>());
```

Comment:
This was, because we not using descartes kinematic interface just the sampler interface now leveraging tesseract kinematics object. Need to see if it is possible to incorporate this into the sampler.